lib/matplotlib/_mathtext_data.py

```diff
@@ -460,640 +460,651 @@
-    'bigoplus'                 : 10753,
-    'leftbrace'                : 123,
-    'rightbrace'               : 125,
-    'jmath'                    : 567,
-    'bigodot'                  : 10752,
-    'preceq'                   : 8828,
-    'biguplus'                 : 10756,
-    'epsilon'                  : 949,
-    'vartheta'                 : 977,
-    'bigotimes'                : 10754,
-    'guillemotleft'            : 171,
-    'ring'                     : 730,
-    'Thorn'                    : 222,
-    'guilsinglright'           : 8250,
-    'perthousand'              : 8240,
-    'macron'                   : 175,
-    'cent'                     : 162,
-    'guillemotright'           : 187,
-    'equal'                    : 61,
-    'asterisk'                 : 42,
-    'guilsinglleft'            : 8249,
-    'plus'                     : 43,
-    'thorn'                    : 254,
-    'dagger'                   : 8224,
-    'increment'                : 8710,
-    'smallin'                  : 8714,
-    'notsmallowns'             : 8716,
-    'smallowns'                : 8717,
-    'QED'                      : 8718,
-    'rightangle'               : 8735,
-    'smallintclockwise'        : 8753,
-    'smallvarointclockwise'    : 8754,
-    'smallointctrcclockwise'   : 8755,
-    'ratio'                    : 8758,
-    'minuscolon'               : 8761,
-    'dotsminusdots'            : 8762,
-    'sinewave'                 : 8767,
-    'simneqq'                  : 8774,
-    'nlesssim'                 : 8820,
-    'ngtrsim'                  : 8821,
-    'nlessgtr'                 : 8824,
-    'ngtrless'                 : 8825,
-    'cupleftarrow'             : 8844,
-    'oequal'                   : 8860,
-    'rightassert'              : 8870,
-    'rightModels'              : 8875,
-    'hermitmatrix'             : 8889,
-    'barvee'                   : 8893,
-    'measuredrightangle'       : 8894,
-    'varlrtriangle'            : 8895,
-    'equalparallel'            : 8917,
-    'npreccurlyeq'             : 8928,
-    'nsucccurlyeq'             : 8929,
-    'nsqsubseteq'              : 8930,
-    'nsqsupseteq'              : 8931,
-    'sqsubsetneq'              : 8932,
-    'sqsupsetneq'              : 8933,
-    'disin'                    : 8946,
-    'varisins'                 : 8947,
-    'isins'                    : 8948,
-    'isindot'                  : 8949,
-    'varisinobar'              : 8950,
-    'isinobar'                 : 8951,
-    'isinvb'                   : 8952,
-    'isinE'                    : 8953,
-    'nisd'                     : 8954,
-    'varnis'                   : 8955,
-    'nis'                      : 8956,
-    'varniobar'                : 8957,
-    'niobar'                   : 8958,
-    'bagmember'                : 8959,
-    'triangle'                 : 9651,
-    'iiiint'                   : 10764,
-    'amalg'                    : 10815,
-    'merge'                    : 10837,
```

Comment:
There is one fewer line in the new code, is something missing? (perhaps one of the things added by a recent PR?)

Reply:
Well spotted! I think that there is a duplicate in the above list.

I current main `len(tex2uni)` is 632 which is 1107-476+1.

Reply:
And it wasn't that hard to find...

``` 
    'nsqsubseteq'              : 8930,
    'nsqsupseteq'              : 8931,
```

Reply:
...but isn't it different: `nsqSUBseteq` and `nsqSUPseteq`(?)

Reply:
I think this is the duplicate entry in main:
 
https://github.com/matplotlib/matplotlib/blob/56732ffcf7a8eb2f4cdbe6835fdc985207c44e09/lib/matplotlib/_mathtext_data.py#L680

https://github.com/matplotlib/matplotlib/blob/56732ffcf7a8eb2f4cdbe6835fdc985207c44e09/lib/matplotlib/_mathtext_data.py#L1098

Reply:
Yeah, you right. I actually read it a few times...